Library/Homebrew/test/cleaner_spec.rb

```diff
@@ -136,6 +136,26 @@
 
       expect(file).not_to exist
     end
+
+    it "removes 'info/**/dir' files except for 'info/<name>/dir'" do
+      file = f.info/"dir"
+      arch_file = f.info/"i686-elf/dir"
+      name_file = f.info/"#{f.name}/dir"
```

Comment:
```suggestion
      name_file = f.info/f.name/"dir"
```

Reply:
I guess it shows that I'm neither a Ruby hacker nor a Homebrew dev. xD

I'll try these suggestions out. :)